src/certifier_tests.cc

```diff
@@ -38,158 +38,153 @@ DEFINE_string(ask_cert_file_name, "ask_cert_file.bin", "ask cert");
 DEFINE_string(vcek_cert_file_name, "vcek_cert_file.bin", "vcek cert");
 
 // Encryption and support tests
-extern bool test_random(bool print_all);
```

Comment:
All these deleted `extern bool` prototypes are now replaced, properly, with `#include .h` files, where-in these prototypes are defined.